### DISCUSSION TITLE
欢迎来到SmartCodable讨论！

### DISCUSSION THREAD

**Discussion by intsig171** [intsig171/SmartCodable#33] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jackjiang** [comment]
对于如图所示，应该如何处理呢？我需要使用let声明字段；
<img width="1094" alt="截屏2025-04-25 15 21 18" src="https://github.com/user-attachments/assets/55be13e0-8b36-4975-855d-8eeab9c9d1c2" />

**Reply by jackjiang** [reply]
@intsig171
